utils.go

```diff
@@ -95,3 +96,6 @@ func isYamlFile(filename string) bool {
 	ext := filepath.Ext(filename)
 	return ext == ".yaml" || ext == ".yml"
 }
+
+var regexpIndex, _ = regexp.Compile("^[0-9]{3}_")
```

Comment:
did you consider not having the underscore as part of the prefix match? what if people just want to write or generate: 010.yaml ?

ps. you could use regexp.Must here

Reply:
I didn't know about the MustCompile - good point.
```suggestion
var regexpIndex = regexp.MustCompile("^[0-9]{3}_")
```

I did consider that people might want to have migrations without descriptions, and they can. Currently it would just be 010_.yaml. Alternatively, the match could end with anything other than a digit as opposed to an underscore.
```suggestion
var regexpIndex = regexp.MustCompile("^[0-9]{3}[^[:digit:]]")
```